fbot_world_bringup/nodes/pose.py

```diff
@@ -119,21 +147,21 @@ def getPose(self, req : GetPose.Request, res : GetPose.Response):
     '''
 
```

Comment:
```suggestion

class_name = req.class if req.class != 'None' else 'targets'
```
Add the default value "targets" as an alternative in case no class was set to the request.